src/dom-to-react/index.js

```diff
@@ -1,5 +1,23 @@
 import React from 'react';
 
+const hyphen2CamelCase = (str) =>  str.replace(/-([a-z])/gi,(s, group) =>  group.toUpperCase());
+const style2object = (styleString) => styleString.split(';').filter(s => s.length).reduce((a, b) => {
+  const keyValue = b.split(':');
+  a[hyphen2CamelCase(keyValue[0])] = keyValue[1];
+  return a;
+}, {});
+
+const poorlyNamedAttributes = {
```

Comment:
It seems to me there are more attributes as in this list.
Would it be possible to import/require this data from React directly?

The list seems to be maintained here:
https://github.com/facebook/react/blob/baff5cc2f69d30589a5dc65b089e47765437294b/packages/react-dom/src/shared/possibleStandardNames.js

It seems in previous versions of react-dom this list has been maintained here: `src/shared/hooks/possibleStandardNames.js` which makes it impracticable to require/import this file.

I think currently it might be the best idea to copy the file into this repo and require it?

Your thoughts on this are much appreciated.

Reply:
I think copying the file is the simplest solution. I m gonna go with that then. 

I want to see if it's possible to not copy the file, but somehow import it via a script during build (tho i am hesitant to do that, and copying might just be the simpler method, albeit if the upstream react changes the list, it's hard to know and so introduce subtle bugs later - but i guess cross that bridge when it happens).